src/test/config.sol

```diff
@@ -78,9 +87,50 @@ contract Config {
     uint256 constant MILLION    = 10 ** 6;
     uint256 constant BILLION    = 10 ** 9;
 
+    uint256 constant monthly_expiration = 4 days;
+    uint256 constant weekly_expiration  = 30 days;
+
+    SpellValues  spellValues;
     SystemValues afterSpell;
 
-    function setCollateralValues() public {
+    function setValues(address chief) public {
+        //
+        // Values for spell-specific parameters
+        //
+        spellValues = SpellValues({
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         1644002846,        // use get-created-timestamp.sh if deployed
+            previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
+            office_hours_enabled:           true,              // true if officehours is expected to be enabled in the spell
```

Comment:
I think office hours could be turned off (as per Doc)

Reply:
Yes!

Reply:
Done.